src/components/caculator.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import Button from './button';
+import Result from './result';
+
+const numbersAndSigns = [
+  'AC',
+  '+/-',
+  '%',
+  '÷',
+  '7',
+  '8',
+  '9',
+  'x',
+  '4',
+  '5',
+  '6',
+  '-',
+  '1',
+  '2',
+  '3',
+  '+',
+  '0',
+  '.',
+  '=',
+];
+
+const Calculator = () => (
```

Comment:
- [ ] Kindly make this component class-based. This is what the requirements demand.

Reply:
I tried but I'm getting an error from react saying Component should be written as a pure function react/prefer-stateless-function